lib/read-theme.js

```diff
@@ -1,21 +1,24 @@
 var Promise = require('bluebird'),
-    _       = require('lodash'),
-    os     = require('os'),
-    hbs     = require('express-hbs'),
-    path    = require('path'),
-    pfs     = require('./promised-fs'),
+    _ = require('lodash'),
```

Comment:
@kevinansfield You can ignore the changes till `extractCustomTemplates` starts, just indentions and lint errors.